www/src/pages/updates/notes/2019-12-17.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: December 17, 2019
+---
+
+This year the Design System turned its attention to our native apps. With help from a number of iOS and Android engineers we were able to hit our component targets and now have near parity across platforms. We also built a lot of other fun stuff along the way.
+
+## 📦 Shipped
+
+We added **31** components to Thumbprint over the past year.
+
+-   iOS: **15**
+-   Android: **11**
+-   React: **5**
+
+## 📈 Growing team
+
+-   Lucas Smith joined us in late 2019 as a designer/developer to help us identify what we can improve and new areas to explore. Coming from Airbnb he's had years of design system experience and we're grateful to have him.
+-   To help us complete our native goals Mallika P. led the efforts for the Android team and Kevin G. on the iOS side.
+
+## 🧰 Projects
+
+-   **Thumbprint open source**: To give back to the design system community and assist with recruiting we open-sourced our codebase in early 2019.
+-   **thumbprint.design refresh**: After our initial rollout in 2018 we updated the look and feel to better represent Thumbtack. Look for more updates coming in 2020.
+-   **The Great Alignment**: To ensure consistency we aligned component names across all our platforms. This meant renaming many components and carefully migrating the changes.
+-   **Responsive images**: We built a robust responsive image component, incorporating lazy loading and our image service, to send the best possible image size to our web users.
+-   **Figma plugins**: We created a few plugins to help teams get their work done faster and there many ideas in the pipeline for 2020.
+-   **Native prototyping**: We now have a system for building native experiences for user research and internal testing using HTML/JS/CSS.
+-   **Figma icons**: Icons in the Thumbprint Icons codebase are now pulled in directly from Figma.
+-   **Illustrations migration**: We worked with the Brand team to replace many old illustrations with shiny new ones.
+-   **TypeScript migration**: Most Thumbprint web components are now using TypeScript to increase consistency and code reliability.
+
+## 🛠️ Tools
+
+-   [Image Service Exporter for Native](https://images.thumbprint.design/tools/image-exporter/)
+-   [Image Service Viewer](https://images.thumbprint.design/tools/image-viewer)
+-   [Brand illustration library](https://images.thumbprint.design/tools/illus-library/index.html)
+-   [New icons UI](https://icons.thumbprint.design)
+
+## 🤗 Thanks
+
+And a special thanks to all the designers and developers who contributed to our team this year, including…
```

Comment:
Rather than listing everyone, I think it'd be better to mention that we had 17 people build components in 2019. 

(This names here don't include people that contributed by filling bugs, fixing bugs, or adding features to existing components.)

Reply:
Removed the names and added a line about other contributors.